src/lib/Preview.js

```diff
@@ -771,10 +771,15 @@ class Preview extends EventEmitter {
             this.retryCount = 0;
         }
 
-        // Fetch access tokens before proceeding
-        getTokens(this.file.id, this.previewOptions.token)
-            .then(this.handleTokenResponse)
-            .catch(this.handleFetchError);
+        const couldPreviewOfflineWithoutToken = typeof fileIdOrFile === 'object' && this.options.skipServerUpdate;
```

Comment:
Can you rename this to something starting with is? maybe ```isOfflinePreview```

Reply:
Thank you, done